Rakefile

```diff
@@ -101,7 +101,8 @@ def package
     system(
       "zip", "-q", "-r", "--symlinks",
       "tuist.zip", "tuist",
-      "ProjectDescription.swiftmodule", "ProjectDescription.swiftdoc", "libProjectDescription.dylib", "ProjectDescription.swiftinterface"
+      "ProjectDescription.swiftmodule", "ProjectDescription.swiftdoc", "libProjectDescription.dylib", "ProjectDescription.swiftinterface",
+      File.expand_path("Templates", __dir__)
```

Comment:
Did you double-check that the zipped directory copies its content recursively?

Reply:
It did, but it was nested deeply in the expanded path, I have modified the copy method to fix it.